src/NzbDrone.Core/CustomFormats/CustomFormatCalculationService.cs

```diff
@@ -166,12 +166,7 @@ private List<CustomFormat> ParseCustomFormat(EpisodeFile episodeFile, Series ser
         {
             var releaseTitle = string.Empty;
 
-            if (episodeFile.SceneName.IsNotNullOrWhiteSpace())
-            {
-                _logger.Trace("Using scene name for release title: {0}", episodeFile.SceneName);
-                releaseTitle = episodeFile.SceneName;
-            }
-            else if (episodeFile.OriginalFilePath.IsNotNullOrWhiteSpace())
+            if (episodeFile.OriginalFilePath.IsNotNullOrWhiteSpace())
```

Comment:
Scene name is either the release's name (from the download client), folder name or file name depending on the release. Leaving original file name in means potentially nothing is excluded (always the case for multi-file releases).

Reply:
I think it would be helpful to explain how the logic works with both the original release title and the renamed file when doing its calculations and help clarify for end users what's going on